### ISSUE TITLE
How is it different from christopheradams/elixir_style_guide?

### ISSUE THREAD

**Issue by marcinbiegun** [lexmag/elixir-style-guide#56]
It would be very useful to pinpoint the differences from the most popular https://github.com/christopheradams/elixir_style_guide guide in the readme file.

**Comment by christopheradams** [comment]
I don't believe that there are any significant disagreements between the style guides. That said, if you think there are any differences that should be pinpointed, you're welcome to contribute them here instead: https://github.com/christopheradams/elixir_style_guide#alternative-style-guides
